src/patternfly/components/Button/examples/Button.md

```diff
@@ -383,6 +392,7 @@ Semantic buttons and links are important for usability as well as accessibility.
 | `.pf-m-block` | `.pf-c-button` | Creates a block level button. |
 | `.pf-m-control` | `.pf-c-button` | Modifies for control styles. **Note:** This modifier should only be used when using buttons in the Input Group or Clipboard Copy components. |
 | `.pf-m-expanded` | `.pf-c-button.pf-m-control` | Modifies a control button for the expanded state. |
+| `.pf-m-selected` | `.pf-c-button` | Modifies a control button for the selected state. |
```

Comment:
I'm assuming this can work with any kind of button?

```suggestion
| `.pf-m-selected` | `.pf-c-button` | Modifies a button for the selected state. |
```